exercises/binary_converter.cpp

```diff
@@ -1,7 +1,28 @@
-/*
-  Write a program that given a number as input convert it in binary.
+#include <iostream>
+#include <cmath>
+using namespace std;
 
-  Output:
-  Insert first number: 8
-  The binary number is: 1000
-*/
+long long convertNumber(int);
+
+int main() {
+    int number, binaryCodeNumber;
+    cout << "Enter number: ";
+    cin >> number;
+    binaryCodeNumber = convertNumber(number);
+    cout << number << " in decimal = " << binaryCodeNumber << " in binary" << endl ;
+    return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```